pkg/sshutil/format.go

```diff
@@ -42,11 +42,11 @@ const (
 	//	  Port 60022
 	FormatConfig = FormatT("config")
 
-// TODO: consider supporting "url" format (ssh://USER@HOSTNAME:PORT)
-//
-// TODO: consider supporting "json" format
-// It is unclear whether we can just map ssh "config" into JSON, as "config" has duplicated keys.
-// (JSON supports duplicated keys too, but not all JSON implementations expect JSON with duplicated keys)
+	// TODO: consider supporting "url" format (ssh://USER@HOSTNAME:PORT).
+	//
+	// TODO: consider supporting "json" format.
+	// It is unclear whether we can just map ssh "config" into JSON, as "config" has duplicated keys.
+	// (JSON supports duplicated keys too, but not all JSON implementations expect JSON with duplicated keys).
```

Comment:
This unrelated change should in another commit. If we don't want prs with multiple commit it should be in another pr.

Reply:
Yeah, this was kind of forced on me by `gofmt`, and then I had to add the trailing `dot` to keep the linter happy.

I need to investigate why this wasn't get flagged by CI before.

I agree that it should have been a separate PR. Requiring commits to be squashed is not a hard rule; we make exceptions when there are good reasons for it.